### DISCUSSION TITLE
Preferred directory structure

### DISCUSSION THREAD

**Discussion by quekyj** [intel/rohd-website#3] [General]
Any preferred directory structure to hold all different sites?

**Comment by mkorbel1** [comment]
I don't have a strong preference as long as it's well organized.

I'm thinking pointers, clones, submodules, etc. of the rest of the ROHD ecosystem (rohd, rohd-vf, rohd-cosim, rohd-hcl) could be in one directory?

Previously in https://github.com/intel/rohd/pull/221 you had all the website stuff in `doc/website`, so probabaly putting that into `website` is a reasonable directory?  Or should it be in the root?

We'll also have other "stuff" in this repository, like minutes for the ROHD Forum, maybe blog-type posts, etc.  Should those be inside the the same directory as the website configuration or should content be distributed?

**Reply by quekyj** [reply]
It seems that GitHub web only allow you to put either `/` or `/docs`?
